classpath/java/util/Properties.java

```diff
@@ -16,7 +16,7 @@
 import java.io.IOException;
 import java.io.Reader;
 
-public class Properties extends Hashtable {
+public class Properties extends Hashtable<Object, Object> {
```

Comment:
This is an unrelated change, basically this was an uninitialized generic and I just initialized it.  It would have defaulted to Object anyway.